src/main/java/frc/robot/subsystems/swerve/SwerveContants.java

```diff
@@ -4,15 +4,20 @@
 
 public class SwerveContants {
 
-    public final static double FALCON_MAX_SPEED = 0;
-    public final static double FALCOM_MAX_ANGULAR_VELOCITY = 0;
-    public final static double GEAR_RATIO = 0;
-    public final static double WHEEL_CIRCUMFERENCE = 0;
-
-    public final static Translation2d FRONT_RIGHT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d FRONT_LEFT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d BACK_RIGHT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d BACK_LEFT_LOCATION = new Translation2d(0, 0);
+    public final static double GEAR_RATIO_DRIVE = 6.75;
+    public final static double GEAR_RATIO_ANGLE = 12.6;
+    public final static double WHEEL_CIRCUMFERENCE = 0.314;
+    public final static double WHEEL_RADIUS = 0.05;
```

Comment:
Why is there no unit? I guess you it is cm, and then I would also advise that you use the official measurement which is 2 inches (and to make the code clear, use the WPILib conversions class from inches to cm).

Reply:
what's 2 inches? the radius?

Reply:
@RBD120307 It's the radius, yes